snippets/stage-autocompleter.json

```diff
@@ -303,16 +303,30 @@
     "prefix": "$out",
     "body": [
       "/**",
-      " * s3Url: A S3 URL to save the data.",
-      " * atlas: Parameters to save to Atlas.",
+      " * Use any one of the following:",
+      " * s3: Parameters to save the data to S3.",
+      " * atlas: Parameters to save the data to Atlas. Example:",
+      " * {",
+      " *   atlas: {",
+      " *     db: 'string',",
+      " *     coll: 'string',",
+      " *     projectId: 'string',",
+      " *     clusterName: 'string'",
+      " *   }",
+      " * }",
       " */",
       "\\$out: {",
-      "  s3: '${1:s3url}',",
-      "  atlas: {",
-      "    db: '${2:db}',",
-      "    coll: '${3:coll}',",
-      "    projectId: '${4:projectId}',",
-      "    clusterName: '${5:clusterName}'",
+      "  s3: {",
+      "    bucket: '${1:string}',",
+      "    region: '${2:string}',",
+      "    filename: '${3:string}',",
+      "    format: {",
+      "      name: '${4:string}',",
+      "      maxFileSize: '${5:bytes}',",
+      "      maxRowGroupSize: '${6:string}',",
+      "      columnCompression: '${7:string}'",
+      "    },",
+      "    errorMode: '${8:string}'",
```

Comment:
Shouldn't this change already be there as it was done long back? If not then I probably should not include it here as well. And what would be the reason for that?

Reply:
Everything is fine here. This file is being generated from the values that come from `'@mongodb-js/mongodb-constants'`. You bumped the version of this package therefore you can see new snippets added here.

[snippets/stage-autocompleter.json](https://github.com/mongodb-js/vscode/blob/main/scripts/update-snippets.ts#L77-L80)